CLI breaks without globally install aws-cdk-lib

### I tried this:

```
wing test main.w
```

or 

```
wing compile -t tf-aws main.w
```

### This happened:

```
Error: Cannot find module 'aws-cdk-lib'
Require stack:
- /Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/target-awscdk/app.js
- /Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/target-awscdk/index.js
- /Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/shared-aws/function.js
- /Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/shared-aws/index.js
- /Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/index.js
- /Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/dist/commands/test.js
- /Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/dist/cli.js
- /Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/bin/wing
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/target-awscdk/app.js:32:26)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/target-awscdk/index.js:17:14)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/shared-aws/function.js:6:25)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/shared-aws/index.js:17:14)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/index.js:33:28)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/dist/commands/test.js:33:15)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at /Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/dist/cli.js:49:112
    at async Command.<anonymous> (/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/dist/cli.js:49:30) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/target-awscdk/app.js',
    '/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/target-awscdk/index.js',
    '/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/shared-aws/function.js',
    '/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/shared-aws/index.js',
    '/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/node_modules/@winglang/sdk/lib/index.js',
    '/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/dist/commands/test.js',
    '/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/dist/cli.js',
    '/Users/sebastian/.nvm/versions/node/v18.16.0/lib/node_modules/winglang/bin/wing'
  ]
}
```

### I expected this:

No error

### Is there a workaround?

Downgrade to `0.24.14`

### Component

CLI

### Wing Version

0.24.15

### Node.js Version

v18.16.0

### Platform(s)

MacOS

### Anything else?

_No response_

### Community Notes

<!-- Please keep this note for the community -->
* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

--- a/libs/wingsdk/src/shared-aws/function.ts
+++ b/libs/wingsdk/src/shared-aws/function.ts
@@ -1,6 +1,5 @@
 import { PolicyStatement } from "./types";
 import { IInflightHost } from "../std";
-import { Function as AwsCdkFunction } from "../target-awscdk";
 import { Function as TfAwsFunction } from "../target-tf-aws";
 
 /**
@@ -32,10 +31,6 @@ export class Function {
    */
   public static from(host: IInflightHost): IAwsFunction | undefined {
     if (host instanceof TfAwsFunction) {
-      return host;
-    }
-
-    if (host instanceof AwsCdkFunction) {
       return host;
     }
 

